nbs/core.ipynb

```diff
@@ -784,6 +784,7 @@
     "            last_dates = self.last_dates\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #560.            cols = self._get_model_names()  # Hack for IQLoss, think about better way
</pre><p>Can you briefly explain the hack in a comment? I believe this line is needed again because the quantiles are modified in the loss of the model above, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1007/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yes, you're right, it's needed because the quantile names are modified during the call to `predict` when one wishes to predict for a certain quantile, but the col names are then already 'established' during the earlier call to self._get_model_names(), resulting in the incorrect column name if one has specified a quantile different from the previous quantile. I couldn't think of an elegant way of solving this (as it seems not trivial to adjust cols afterwards either - then calling this function again migh be easiest.)